Building docker image with aarch64 build fails due to mismatching checksum

After upgrading to version `0.30.1` building docker image with arm architecture causes the build to fail with 

`npm ERR! Downloaded dprint zip checksum did not match the expected checksum (Actual: 69875e044b831ba34df72fbc9009f269909cf72d18485d876690ee9f51a5dfd5, Expected: c8b45ca831b742c3555060e4895e4adcd03894a2ad017f0fb9919adb99060197)` 

Seems like its expecting the `x86_64` checksum even when using `aarch64`

--- a/crates/dprint/Cargo.toml
+++ b/crates/dprint/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "dprint"
-version = "0.30.2"
+version = "0.30.3"
 authors = ["dprint code formatting <david@dprint.dev>"]
 categories = ["development-tools"]
 edition = "2021"

--- a/deployment/npm/install.js
+++ b/deployment/npm/install.js
@@ -140,7 +140,11 @@ function verifyZipChecksum() {
           return info.checksums["darwin-x86_64"];
         }
       default:
-        return info.checksums["linux-x86_64"];
+        if (os.arch() === "arm64") {
+          return info.checksums["linux-aarch64"];
+        } else {
+          return info.checksums["linux-x86_64"];
+        }
     }
   }
 }

